R/Prediction_operators.R

```diff
@@ -58,6 +60,8 @@ getPredictionProbabilities = function(pred, cl) {
   } else if (ttype == "cluster") {
     y = pred$data[, stri_detect_regex(cns, "prob\\.")]
     colnames(y) = seq_col(y)
+  } else if (ttype == "surv") {
+    y = pred$data[, stri_detect_regex(cns, "prob\\.")]
```

Comment:
`stri_startswith_fixed()`? (although two lines above there is the same "mistake")

Reply:
ok